src/Keyboard.elm

```diff
@@ -42,7 +42,7 @@ keyCode =
 -- MOUSE EVENTS
 
 
-{-| Subscribe to all key presses.
+{-| Subscribe to all presses of keys that normally produce a character value.
```

Comment:
Let's let @evancz decide about the exact wording if he agrees to a documentation fix here at all.